vector-im/element-android
# #5123: @room tag suggestion

## Type of change


## Content


Adding possibility to notify everyone in a room using "@room" tag. The ability to notify everyone depends on the user permission level in the room.

## Motivation and context


See #5123 

## Screenshots / GIFs


TODO

## Tests



- Go to a room screen where you have enough permission to notify everyone (e.g. a room you created)
- Type "@" 
- Check that "@room" appears in the suggestion list in a dedicated section 
- Check the suggestion list corresponds to what is typed
- Select the "@room" tag
- Check the tag appears as a "chip"
- Write a message
- Send the message
- Check all users of the room are notified of the message
- Go to a room screen where you don't have enough permission to notify everyone
- Type "@" 
- Check that "@room" does not appear in the suggestion list

## Tested devices

- OS version(s): Android 11

## Checklist
